bundles/org.openhab.core.addon/src/main/java/org/openhab/core/addon/infoproviders/AddonsInfoProvider.java

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.core.addon.infoproviders;
```

Comment:
Please always use singular for package names.
But does this really need a separate package? And does it have to be a public one or could it be moved to `org.openhab.core.addon.internal.xml`?

Reply:
TBH I am not sure if it needs to be public; maybe #3806 needs it? But I will move it to internal here, and we could eventually move it back in #3806..